### DISCUSSION TITLE
Get sponsored for project

### DISCUSSION THREAD

**Discussion by ebrahimraeyat** [xcfem/xc#91] [Ideas]
Hello @lcpt. As you may know, it is possible to get sponsored with GitHub now. Do you think it is useful for project?
I think it is so. For example, I can work on this project with about $1000 in a month. What do you think?

**Comment by lcpt** [comment]
Hi @ebrahimraeyat 
This week I've started the process to get sponsored in GitHub. We'll see.
Anyway, I think that to get some decent revenue we'll need to make things a lot easier for users ([Windows installation](https://ubuntu.com/tutorials/ubuntu-on-windows#1-overview), enhance documentation, IFC interface, steel connections, ...).

**Comment by ebrahimraeyat** [comment]
Hi @lcpt. That's all true, but your project has the high level potentials for all things you mentioned. I can work seriously  on this project if I have only $500 per month. I can create some GUI  and IFC interface for starting. When the project start to grow and thus more people could work with it, probably more revenue comes from more peoples. It is important to prepare minimum things for usual users to allow them to work with the software. Then we can work on additional  features. What is your opinion? Thanks.

**Comment by ebrahimraeyat** [comment]
Also, using FreeCAD as GUI may help us to attract more developer or sponsored.

**Comment by lcpt** [comment]
Hi @ebrahimraeyat.
My opinion is that, if we achieve to get some income from the development we have done, we must care about all the contributors. In the meantime, everyone contributes according to his possibilities.

**Reply by ebrahimraeyat** [reply]
Surely, and I believe that you (and Ana I think), have done the major work and wrote the heavy foundation for XCFEM. I am very happy to be a contributor of this project.
